### ISSUE TITLE
[Header]: Add navigation links that break to two lines

### ISSUE THREAD

**Issue by mariamarrero** [uswds/uswds#1638]
Text in tabs and amount of tabs in main navigation (Header components)

It would be very helpful if one of the Header components has an example of a main navigation where long links wrap to a second line.

In the case of USA.gov's main navigation, it has 7 tabs and each one needs to wrap to a second line so that all can fit. It would be great if there is a version that reflects this structure for websites that have more than 4 topics to display. This will also reduce the amount of customization of the Header components by agencies' development teams.

**Comment by shawnbot** [comment]
Thanks for this, @mariamarrero!

@juliaelman, I think this is the sort of use case that would be easier to create a test for if the navigation were a distinct component.

**Comment by shawnbot** [comment]
For reference, here is what the USA.gov navigation looks like right now:

![image](https://cloud.githubusercontent.com/assets/113896/21323546/da7f2c8a-c5d2-11e6-90c4-b937fdf68f7d.png)

**Comment by mariamarrero** [comment]
Sure thing!

Not sure if this is helpful or not, but something we had to customize
(based on the IA and content needs) is the amount of columns under the mega
menu. We added a 4th column. There was extra space for this, so that was a
plus.

*Maria Marrero*
User Experience Designer, USAGov <http://usa.gov/explore>
Your Guide to Government Information and Services
GSA, Office of Products and Programs
202-494-9522
